src/backend/expungeservice/expunger.py

```diff
@@ -60,6 +60,14 @@ def run(self) -> bool:
                     (charge.disposition.date + relativedelta(years=1), "Time-ineligible under 137.225(1)(b)")
                 )
 
+            if charge.convicted() and charge.case()().get_probation_revoked():
+                eligibility_dates.append(
+                    (
+                        charge.disposition.date + relativedelta(years=10),
+                        "Time-ineligible under 137.225(1)(c). See comments for when there are multiple convictions in a case.",
```

Comment:
This looks like a placeholder, but a reason string should be readable by a user who doesn't need/want to consult in-code content.